pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -291,6 +245,38 @@ func BuildSidecarOutboundVirtualHosts(node *model.Proxy, push *model.PushContext
 		}
 	}
 
+	buildVirtualHost := func(hostname string, vhwrapper istio_route.VirtualHostWrapper, svc *model.Service) *route.VirtualHost {
```

Comment:
seems unrelated change

Reply:
As i mentioned in PR - I did few more refactoring to simplify the code